binding/nodejs/src/cheetah.ts

```diff
@@ -48,17 +52,17 @@ export default class Cheetah {
   /**
    * Creates an instance of Cheetah.
    * @param {string} accessKey AccessKey obtained from Picovoice Console (https://console.picovoice.ai/).
-   * @param {number} endpointDurationSec Duration of endpoint in seconds. A speech endpoint is detected when there is a segment of audio
-   * (with a duration specified herein) after an utterance without any speech in it. Set to `0` to disable endpoint detection.
-   * @param {string} modelPath the path to the Cheetah model (.pv extension)
-   * @param {string} libraryPath the path to the Cheetah dynamic library (.node extension)
+   * @param options Optional configuration arguments.
+   * @param {string} options.modelPath the path to the Cheetah model (.pv extension)
+   * @param {string} options.libraryPath the path to the Cheetah library (.node extension)
+   * @param {number} options.endpointDurationSec Duration of endpoint in seconds. A speech endpoint is detected when there is a
+   * chunk of audio (with a duration specified herein) after an utterance without any speech in it. Set to `0`
+   * to disable endpoint detection.
+   * @param {boolean} options.enableAutomaticPunctuation Flag to enable automatic punctuation insertion.
    */
   constructor(
     accessKey: string,
-    endpointDurationSec: number,
-    modelPath?: string,
-    libraryPath?: string
-  ) {
+    options: CheetahConfig = {}) {
```

Comment:
can we run formatter in cheetah as well?

Reply:
Actually I did, i guess WebStorm formatter follows a bit different style here

Reply:
oh ok